From ce89c6dd0debf134c09f367ea26df4e340b81e73 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:06:12 -0400
Subject: [ethernet] qlcnic: Enhance driver message in failed state

Message-id: <1408626422-3941-89-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88637
O-Subject: [PATCH RHEL 7.1 088/138] qlcnic: Enhance driver message in failed state.
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Sucheta Chakraborty <sucheta.chakraborty@qlogic.com>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id 2a355aecd27f88d26ba8213b4e55123522ef5eae

Signed-off-by: Sucheta Chakraborty <sucheta.chakraborty@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
index 5942f5b..a6b9a7f 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_main.c
@@ -2447,8 +2447,8 @@ qlcnic_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
   if (err) {
    switch (err) {
    case -ENOTRECOVERABLE:
-    dev_err(&pdev->dev, "Adapter initialization failed due to a faulty hardware. Please reboot\n");
-    dev_err(&pdev->dev, "If reboot doesn't help, please replace the adapter with new one and return the faulty adapter for repair\n");
+    dev_err(&pdev->dev, "Adapter initialization failed due to a faulty hardware\n");
+    dev_err(&pdev->dev, "Please replace the adapter with new one and return the faulty adapter for repair\n");
     goto err_out_free_hw;
    case -ENOMEM:
     dev_err(&pdev->dev, "Adapter initialization failed. Please reboot\n");
-- 
1.7.1